reports/validate_reports.py

```diff
@@ -41,6 +41,7 @@ def validate_files(output_dir: str, schema_dir: str, files: list) -> None:
     "3_routes_changed.json",
     "3_stops_changed.json",
     "4_guideline_checks_schedule.json",
+    # "4_guideline_checks_rt.json",
```

Comment:
is this commented because it currently fails validation?

Reply:
Well technically it _passes_ validation because that is commented out!

Oof. Thanks for catching that. Yes I believe it will fail if I uncomment it out, will update and re-run validation.

Reply:
good catch!

Reply:
I updated this, ran it and it passes (I had already made the fix to validation issues experienced previously)